cmd/antrea-agent/options.go

```diff
@@ -383,17 +385,33 @@ func (o *Options) setK8sNodeDefaultOptions() {
 	}
 
 	if features.DefaultFeatureGate.Enabled(features.FlowExporter) {
-		if o.config.FlowCollectorAddr == "" {
-			o.config.FlowCollectorAddr = defaultFlowCollectorAddress
+		if o.config.FlowExporter.FlowCollectorAddr == "" {
+			o.config.FlowExporter.FlowCollectorAddr = defaultFlowCollectorAddress
+			if o.config.FlowCollectorAddr != "" {
+				klog.InfoS("The FlowCollectorAddr option is deprecated, please use flowExporter.FlowCollectorAddr instead")
```

Comment:
```suggestion
				klog.InfoS("The flowCollectorAddr option is deprecated, please use flowExporter.flowCollectorAddr instead")
````

same for others below

Reply:
Fixed. Thanks.